### ISSUE TITLE
No man, info or help page to list wmail terminal commands

### ISSUE THREAD

**Issue by SamuelMiller** [Thomas101/wmail#594]
Hello,
Thanks for the great application. I want to create a custom send-to action in XFCE's file manager so I am able right-click on an item in the file manager (e.g., Thunar) and send it directly as an attachment directly through Wmail. However, without a man/help page for the wmail application, I am unable to find the correct command to do so. 

Thanks, Sam

**Comment by Thomas101** [comment]
Yeah I agree it should be there now that WMail is starting to support command line args. In the meantime there's some info on command line args [here](https://github.com/Thomas101/wmail/wiki/FAQs#command-line-arguments).

Nothing on sending attachments straight in, so that's probably something to look at separately.
